docs/components/FileInputView.jsx

```diff
@@ -25,7 +47,7 @@ export default function FileInputView() {
                   if (progressPercent !== 100) {
                     progressPercent += 5;
                     progress(progressPercent);
-                  } else if (this.state.error) {
+                  } else if (isError()) {
```

Comment:
if this is always false, do do we need this `else if` here?

Reply:
The existing example `FileInput` demo page is currently throwing an error on this line https://clever.github.io/components/#/components/file-input

![screen shot 2018-09-05 at 11 42 46 am](https://user-images.githubusercontent.com/1817/45114097-eda61100-b100-11e8-9134-5a79e4bb809b.png)

I believe this function exists mostly as documentation. You can see it in the code example on the page, and when you mock "upload" a file, this function fills the progress indicator over time.

Unfortunately, in combination with the `setInterval()` call and the `if (progressPercent !== 100)` check, it means that the page will continuously throw errors forever in a loop, and the success icon will never be displayed.

I thought I would fix up this error while I was at it, but to be honest, this change is not even related to my changes nor the second example I created down lower on the page. I can back this out if you like.

Reply:
oh i see, it's purely to show example code. Fine to leave it as is!

If you wanted to, you could add a toggle on the documentation to 'show error state' so they can see what happens in this example, though it wouldn't be too exciting haha